src/config.ts

```diff
@@ -4,9 +4,11 @@ export const defaultConfig = overrideWithEnv(getDefaultConfig())
 export const configHelp = getConfigHelp()
 
 export function getDefaultConfig(): PluginsServerConfig {
+    const isTestEnv = process.env.NODE_ENV === 'test'
```

Comment:
Ah, we probably should actually use `NODE_ENV` instead of `areWeTestingWithJest()`'s `JEST_WORKER_ID` 🤔

Reply:
yeah, fixed!